tools/chorctl/cmd/repl_getReplicationStatus.go

```diff
@@ -51,9 +51,9 @@ chorctl repl get -f main -t follower -u admin -b bucket1`,
 		// io.Writer, minwidth, tabwidth, padding int, padchar byte, flags uint
 		w := tabwriter.NewWriter(os.Stdout, 10, 1, 5, ' ', 0)
 
-		fmt.Fprintln(w, "USER\tBUCKET\tFROM\tTO\tCREATED AT\tPAUSED\tINIT DONE\tEVENTS\tLAST EMITTED AT\tLAST PROCESSED AT\tAGENT URL\tSWITCH STATUS\tTO BUCKET\tINIT DONE AT")
+		fmt.Fprintln(w, "USER\tBUCKET\tFROM\tTO\tCREATED AT\tPAUSED\tINIT DONE\tEVENTS\tLAST EMITTED AT\tLAST PROCESSED AT\tAGENT URL\tHAS SWITCH\tTO BUCKET\tINIT DONE AT")
```

Comment:
Don't you want to move these to `mappers.go`, where the other table printing functions are located?

Reply:
done